website/docs/migrating-to-0.7.0.mdx

```diff
@@ -84,24 +84,26 @@ Before:
 
 After:
 
-```svelte
-<script context="module" lang="ts">
-  import { browser } from '$app/env'
-  import { GQL_FirstQuery } from '$houdini'
-  import type { Load } from '@sveltejs/kit'
+```ts
+// +page.ts
+import { GQL_FirstQuery } from '$houdini'
+import type { Load } from './$types'
 
-  export const load: Load = async (event) => {
-    await GQL_FirstQuery.fetch({ event })
-    return {}
-  }
-</script>
+export const load: Load = async (event) => {
+  await GQL_FirstQuery.fetch({ event })
```

Comment:
we should probably push people to use `load_FirstQuery` so they don't accidently use a global store and break pre-fetching

Reply:
Totally, great catch 👍